GoogleTestAdapter/Packaging.GTA/source.extension.vsixmanifest

```diff
@@ -6,7 +6,7 @@
     <Description xml:space="preserve">Adds support for the C++ unit testing framework Google Tests.</Description>
     <MoreInfo>https://github.com/csoltenborn/GoogleTestAdapter</MoreInfo>
     <License>LICENSE.txt</License>
-    <Icon>Resources\gtest-logo.gif</Icon>
+    <Icon>Resources\Icons\Icon.ico</Icon>
```

Comment:
This has to be the path inside VSIX as is generated, and Icon.ico is copied to Resources\Icons\Icon.ico, so this is OK.